packages/wallet-sdk/src/core/communicator/Communicator.ts

```diff
@@ -96,7 +96,7 @@ export class Communicator {
       .then((message) => {
         this.postMessage({
           requestId: message.id,
-          data: { version: LIB_VERSION },
+          data: { version: LIB_VERSION, appFullUrl: window.location.href },
```

Comment:
Could this be spoofed by a malicious dapp? If so, is it possible to use window.referrer?

Reply:
i thought you can get this already via `window.opener` or `event.origin`?
cc @willnewman-cb @fan-zhang-sv